src/main.rs

```diff
@@ -1,4 +1,4 @@
-//! As per description, is a reimpl of nixos-rebuild with room for opinionation.
+//! As per description, is a reimplementation of nixos-rebuild with room for opinionatedness.
```

Comment:
For people that have English as a first language, abbreviations can be challenging. Also "opinionation" is not a word :wink: